core/trino-main/src/main/java/io/trino/operator/aggregation/ClassificationMissRateAggregation.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Iterator;
 
 @AggregationFunction("classification_miss_rate")
-@Description("Computes miss-rate for precision-recall curves")
+@Description("Computes miss-rate (false-negative rate) for binary classification")
```

Comment:
We may want to change the name of the function to `classification_false_negative_rate`